### ISSUE TITLE
Select appropriate petitions to reflect the range of debate

### ISSUE THREAD

**Issue by kevinmarks** [indicativevote/indicative-vote#5]
May need to create new petitions for this

**Comment by kevinmarks** [comment]
please add petition IDs or URLs here. If I should remove one say so

**Comment by aspiers** [comment]
Yes it's not just about creating new petitions, it's about carefully selecting the right ones (e.g. see #10), and also clearly distinguishing between process and outcome (see #9).

Another question is whether we want to mirror Commons debates / divisions as closely as possible, even if what the MPs are doing doesn't end up being entirely logical.

**Comment by aspiers** [comment]
The FT are doing something similar: 

https://survey.ft.com/jfe/form/SV_0U34PUj8vQyC4XX

We probably should steal options from here.

**Comment by kevinmarks** [comment]
awaiting approval for https://petition.parliament.uk/petitions/251971
